tests/conftest.py

```diff
@@ -125,16 +126,17 @@ def parallel(item):
             # run on the current machine
             continue
 
+        pyversion = sys.executable
         # Only spew tracebacks on rank 0.
         # Run xfailing tests to ensure that errors are reported to calling process
         if item.cls is not None:
             testname = "%s::%s::%s" % (item.fspath, item.cls.__name__, item.name)
         else:
             testname = "%s::%s" % (item.fspath, item.name)
-        args = ["-n", "1", "python", "-m", "pytest", "--runxfail", "-s",
+        args = ["-n", "1", "%s" % pyversion, "-m", "pytest", "--runxfail", "-s",
```

Comment:
why

```
"%s" % pyversion
```

and not just

```
pyversion
```

?

Reply:
@FabioLuporini I fixed this. I forgot that there from a previous attempt. Thanks. As I can see GPU-test passes. Feel free to merge or review again after it gets green.